CommunityProject/Assets/_Game/Scripts/Editor/Windows/LevelSelectionEditorWindow.cs

```diff
@@ -0,0 +1,117 @@
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+
+namespace BoundfoxStudios.CommunityProject.Editor.Windows.LevelSelectionEditorWindow
```

Comment:
```suggestion
namespace BoundfoxStudios.CommunityProject.Editor.Windows
```

Namespaces sind nur auf Ordner-Ebene, d.h. es endet hier beim Ordner "Windows" :)